qcodes/station.py

```diff
@@ -355,11 +378,22 @@ def update_load_instrument_methods():
                         self.load_instrument, identifier=instrument_name))
                     self._added_methods.append(method_name)
                 else:
-                    log.warning(f'Invalid identifier: ' +
-                                f'for the instrument {instrument_name} no ' +
-                                f'lazy loading method {method_name} could ' +
+                    log.warning(f'Invalid identifier: '
+                                f'for the instrument {instrument_name} no '
+                                f'lazy loading method {method_name} could '
                                 'be created in the Station.')
-        self._config = YAML().load(config)
+        yaml = YAML().load(config)
```

Comment:
Could you add a comment here explaining why this uses the template for future reference